### PR TITLE
chore(ui): add `initialOptions` prop in tag suggestion component

### PR DESCRIPTION
**Pull request by Sachin-chaurasiya**
### Describe your changes:

I worked on adding `initialOptions` prop in the tag suggestion component so that it can render the initial values options with proper styling and label.

#
### Type of change:

- [x] Improvement

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/shared/TagSuggestion.tsx [hunk 1]
```diff
@@ -82,6 +85,7 @@ const TagSuggestion: React.FC<TagSuggestionProps> = ({
     <AsyncSelectList
       defaultValue={value?.map((item) => item.tagFQN) || []}
       fetchOptions={isGlossaryType ? fetchGlossaryList : fetchTagsElasticSearch}
+      initialData={initialData}
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/shared/TagSuggestion.tsx:88]: @Sachin-chaurasiya what's difference between initialData & defaultValue?

Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/shared/TagSuggestion.tsx:88]: @chirag-madlani , defaultValue is for the select default values and initialData is for options.

Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/shared/TagSuggestion.tsx:88]:
```suggestion
      initialOptions={initialOptions}
```

Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/shared/TagSuggestion.tsx:88]: @chirag-madlani , cool, that would be more helpful, thanks!

**Comment by Sachin-chaurasiya** [comment]
@chirag-madlani, I have addressed the comments, can you re-review them?

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**